bindings/python/crocoddyl/multibody/contacts/contact-6d-loop.cpp

```diff
@@ -0,0 +1,245 @@
+          ":param gains: gains of the contact model (default "
+          "np.matrix([0.,0.]))"))
+      .def("calc", &ContactModel6DLoop::calc, bp::args("self", "data", "x"),
+           "Compute the 6D contact Jacobian and drift.\n\n"
+           "The rigid contact model throught acceleration-base holonomic "
+           "constraint\n"
+           "of the contact frame placement.\n"
+           ":param data: contact data\n"
+           ":param x: state point (dim. state.nx)")
+      .def("calcDiff", &ContactModel6DLoop::calcDiff,
+           bp::args("self", "data", "x"),
+           "Compute the derivatives of the 6D contact holonomic constraint.\n\n"
+           "The rigid contact model throught acceleration-base holonomic "
+           "constraint\n"
+           "of the contact frame placement.\n"
+           "It assumes that calc has been run first.\n"
+           ":param data: cost data\n"
+           ":param x: state point (dim. state.nx)")
+      .def("updateForce", &ContactModel6DLoop::updateForce,
+           bp::args("self", "data", "force"),
+           "Convert the Lagrangian into a stack of spatial forces.\n\n"
+           ":param data: cost data\n"
+           ":param force: force vector (dimension 6)")
+      .def("createData", &ContactModel6DLoop::createData,
+           bp::with_custodian_and_ward_postcall<0, 2>(),
+           bp::args("self", "data"),
+           "Create the 6D contact data.\n\n"
+           "Each contact model has its own data that needs to be allocated. "
+           "This function\n"
+           "returns the allocated data for a predefined cost.\n"
+           ":param data: Pinocchio data\n"
+           ":return contact data.")
+      .add_property(
+          "joint1_id",
+          bp::make_function(&ContactModel6DLoop::get_joint1_id,
+                            bp::return_value_policy<bp::return_by_value>()),
+          "reference joint id of the first contact frame")
+      .add_property(
+          "joint1_placement",
+          bp::make_function(&ContactModel6DLoop::get_joint1_placement,
+                            bp::return_value_policy<bp::return_by_value>()),
+          "Placement of the first contact frame")
+      .add_property(
+          "joint2_id",
+          bp::make_function(&ContactModel6DLoop::get_joint2_id,
+                            bp::return_value_policy<bp::return_by_value>()),
+          "reference joint id of the second contact frame")
+      .add_property(
+          "joint2_placement",
+          bp::make_function(&ContactModel6DLoop::get_joint2_placement,
+                            bp::return_value_policy<bp::return_by_value>()),
+          "Placement of the second contact frame")
+      .add_property(
+          "gains",
+          bp::make_function(&ContactModel6DLoop::get_gains,
+                            bp::return_value_policy<bp::return_by_value>()),
+          "contact gains")
+      .def(CopyableVisitor<ContactModel6DLoop>());
+
+#pragma GCC diagnostic pop
```

Comment:
what are you popping here ? I can't see the push

Reply:
I removed this pop